.changeset/heavy-snails-help.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/rest-typings": minor
+---
+
+implemented "update contacts" endpoint to the omnichannel
```

Comment:
```suggestion
Implemented "update contacts" endpoint to the omnichannel
```

Reply:
Think you can also describe a bit more the endpoint, and even put the endpoints short route in here.

(this is a personal pref so if you feel doens't make sense here, feel free to ignore)